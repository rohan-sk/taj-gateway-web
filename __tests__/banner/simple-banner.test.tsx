import { render } from "@testing-library/react";
import SimpleBanner from "../../components/banner/simple-banner.component";

const data = {
  image: "image-8bd59520af0028434a7ee36725f77ef0e608a27a-640x640-png",
  largeImage: "image-8e11aabadd082534345dd320fbd066f78a1be13d-1920x770-png",
};

it("shows simple banner", () => {
  const { asFragment } = render(<SimpleBanner {...data} />);
  expect(asFragment()).toMatchSnapshot();
});
