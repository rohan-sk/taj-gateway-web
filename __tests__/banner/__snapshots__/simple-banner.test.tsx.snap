// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shows simple banner 1`] = `<DocumentFragment />`;
